packages/components/src/navigator/styles.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { css, keyframes } from '@emotion/react';
+
+export const navigatorProviderWrapper = css`
+	/* Prevents horizontal overflow while animating screen transitions */
+	overflow-x: hidden;
+
+	contain: strict;
+`;
+
+const fadeInFromRight = keyframes( {
+	'0%': {
+		opacity: 0,
+		transform: `translateX(50px)`,
+	},
+	'100%': { opacity: 1, transform: 'none' },
+} );
+
+const fadeInFromLeft = keyframes( {
+	'0%': {
+		opacity: 0,
+		transform: `translateX(-50px)`,
+	},
+	'100%': { opacity: 1, transform: 'none' },
+} );
+
+type NavigatorScreenAnimationProps = {
+	isInitial?: boolean;
+	isBack?: boolean;
+	isRTL: boolean;
+};
+
+const navigatorScreenAnimation = ( {
+	isInitial,
+	isBack,
+	isRTL,
+}: NavigatorScreenAnimationProps ) => {
+	if ( isInitial && ! isBack ) {
+		return css``;
+	}
+
+	const animationName =
+		( isRTL && isBack ) || ( ! isRTL && ! isBack )
+			? fadeInFromRight
+			: fadeInFromLeft;
+
+	return css`
+		animation-duration: 0.14s;
+		/* easeInOutCubic */
+		animation-timing-function: cubic-bezier( 0.65, 0, 0.35, 1 );
```

Comment:
Isn't this a better mirror representation of the originally used `easeInOut` easing function:

```suggestion
		animation-timing-function: cubic-bezier( 0.42, 0, 0.58, 1 );
```

Or are we intentionally adding a bit of an acceleration effect by using the cubic version?

Note that this is mostly a question and I don't have strong preferences on the easing we're using.

Reply:
Good point, I actually switched to the standard CSS `ease-in-out` keyword (which represents the same bezier curve that you suggested)

https://github.com/WordPress/gutenberg/pull/56909/commits/5d662f78041410ab97511e9a8582b030d464aa0e